cmd/scollector/collectors/sql_windows.go

```diff
@@ -50,6 +61,7 @@ const (
 )
 
 var (
+	cluster        string
```

Comment:
Name this sqlCluster or something uniqueish. This file is part of a package with many other collectors and we don't want to have any file reserve use of a common name.